C/ds/Dijsktra.c

```diff
@@ -3,8 +3,11 @@
 #include<stdlib.h>
 
 void main(){
-
-	int cost[10][10],distance[10],path[10][10],n,v,p,row,column,min,index=1,i,j;
+	
+	int size;
+	printf("Enter the size of array:");
```

Comment:
User will not understand what is mean by size of array 
Please display the message clearly, what are you taking in form **size**

Reply:
Display something relevant message

Reply:
done @siddharth25pandey 
kindly review it